x/stake/tick_test.go

```diff
@@ -121,14 +121,13 @@ func TestProcessProvisions(t *testing.T) {
 	//panic(fmt.Sprintf("debug total %v, bonded  %v, diff %v\n", p.TotalSupply, p.BondedPool, pool.TotalSupply-pool.BondedPool))
 
 	// initial bonded ratio ~ from 27% to 40% increase for bonded holders ownership of total supply
-	assert.True(t, pool.bondedRatio().Equal(sdk.NewRat(271734723, 671734723)), "%v", pool.bondedRatio())
+	assert.True(t, pool.bondedRatio().Equal(sdk.NewRat(211813022, 611813022)), "%v", pool.bondedRatio())
```

Comment:
Why did the numbers change here, were our operations wrong before or just rounded differently?

Reply:
It is due to the increase in precision associated with embedded rational - we should increase the testing of TestProcessProvisions for further scenarios - I'll open up an issue